src/components/EditTodo/EditTodo.jsx

```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react';
+import { saveChangeTask } from 'service/taskService';
+import { isValidValueInput } from 'helpers/validation';
+import 'components/Todos/style.scss'
```

Comment:
each component must have its own styles, it must not refer to the styles of another component